src/lib/swapper/swappers/ThorchainSwapper/utils/getUniqueAddressSubstring.ts

```diff
@@ -0,0 +1,34 @@
+import { type AssetId, fromAssetId } from '@shapeshiftoss/caip'
+
+export const getUniqueAddressSubstring = (destinationAddress: string, addresses: AssetId[]) => {
```

Comment:
```suggestion
export const getUniqueAddressSubstring = (destinationAddress: string, longTailAssetIds: AssetId[]) => {
```

Reply:
And if you pass `destinationAssetId: AssetId`, instead of an address, there is no need for assetReference checks (`toAssetId()` already lowercases the `assetReference` part)